src/sentry/static/sentry/app/views/settings/incidentRules/triggers/chart/thresholdsChart.tsx

```diff
@@ -224,6 +224,7 @@ export default class ThresholdsChart extends React.PureComponent<Props, State> {
 
   render() {
     const {data, triggers, xAxis} = this.props;
+    data[0].data = data[0].data.splice(-1, 1);
```

Comment:
We shouldn't mutate objects like this because it can break things in react, e.g. lifecycle methods when we compare current to previous props can break. This is also using splice again which returns the items removed.

We should map through `data` (just in case there's multiple series or something) as well.

Something like this:

```suggestion
    const dataWithoutRecentBucket = data.map(({data, ...restOfData}) => ({
    ...restOfData,
    data: data.slice(0, -1),
    }));
```